Doc/library/typing.rst

```diff
@@ -1409,55 +1566,23 @@ These are not used in annotations. They are building blocks for creating generic
 
     .. versionadded:: 3.11
 
-.. data:: Unpack
```

Comment:
Why is this being moved to a different place? It makes the diff harder to read :/

If it needs to be moved, maybe separate it into its own PR?

Reply:
Currently there is a section on generics. I want to change that to a section on "things you can make through dedicated syntax", including the generics and TypeAliasType. But that required moving the odd ones out of the generics section: Unpack, Protocol, and AnyStr. Unpack and Protocol arguably don't belong there regardless, so I can move them out in a separate PR if you like.

Reply:
Thanks, that makes sense!

> Unpack and Protocol arguably don't belong there regardless, so I can move them out in a separate PR if you like.

I think I'd prefer that, if it's okay (sorry if I'm being pedantic 😕)